.github/workflows/nextcloud-branch-deletion.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   delete_namespace_and_buckets:
-    if: inputs.NAMESPACE_NAME != 'nbc-main'
+    if: inputs.NAMESPACE_NAME != 'nbc-main'  || inputs.NAMESPACE_NAME != 'brb-main' || inputs.NAMESPACE_NAME != 'thr-main' 
```

Comment:
Shouldn't this be and not or? This will always be true

Reply:
very very true